README.md

```diff
@@ -15,6 +15,90 @@ To install `odin.api`:
 remotes::install_github("mrc-ide/odin.api", upgrade = FALSE)
 ```
 
+## Usage
+
+```
+docker pull mrcide/odin.api
+docker run -d --rm -p 8001:8001 mrcide/odin.api
```

Comment:
Might be nice to indicate that you can pull individual branches with `docker pull mrcide/odin.api:branch-name` as well, which is what I did to test this branch.